### PR TITLE
New package: way-displays-1.7.1

### PR DESCRIPTION
**Pull request by dv-ar**
#### Testing the changes
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): YES

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-LIBC)

### CODE REVIEW

File: srcpkgs/way-displays/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="wayland-devel yaml-cpp-devel wlroots-devel eudev-libudev-devel"
+short_desc="Auto Manage Your Wayland Displays"
+maintainer="dvar <dvar@tuta.io>"
+license="MIT"
+homepage="https://github.com/alex-courtis/way-displays"
+distfiles="https://github.com/alex-courtis/way-displays/archive/refs/tags/${version}.tar.gz"
+checksum=7a0ed9281e8cd1c0fb1e43b86bfb9d21c6f26af13be8b22b370fd79ab1e73c47
+patch_args="-Np1"
```
Comment (abenson) [inline srcpkgs/way-displays/template:14]: This is the default so it can be omitted here.

File: srcpkgs/way-displays/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+post_install() {
+vlicense LICENSE
```
Comment (abenson) [inline srcpkgs/way-displays/template:17]: Indent these lines with tabs.

File: srcpkgs/way-displays/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'way-displays'
+pkgname=way-displays
+version=1.7.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
```
Comment (abenson) [inline srcpkgs/way-displays/template:6]: Add `wayland-devel` to `hostmakedepends`

File: srcpkgs/way-displays/patches/sanitize_configmk.patch [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+*** 2,5 ****
+  
+! PREFIX ?= /usr/local
+! PREFIX_ETC ?= /usr/local
+  ROOT_ETC ?= /etc
+--- 2,5 ----
+  
+! PREFIX ?= /usr
+! PREFIX_ETC ?= /
+  ROOT_ETC ?= /etc
```
[the remark this answers is not in the document]
Comment (dv-ar) [reply srcpkgs/way-displays/patches/sanitize_configmk.patch:13]: How do i set the variable, I've tried setting the following options in the template without success.
```
make_build_args="PREFIX=/usr PREFIX_ETC=/"
make_install_args="PREFIX=/usr PREFIX_ETC=/"
make_check_args="PREFIX=/usr PREFIX_ETC=/"
```

[the remark this answers is not in the document]
Comment (dv-ar) [reply srcpkgs/way-displays/patches/sanitize_configmk.patch:13]: nevermind...

**Pull request merged**